change/@fluentui-react-button-33f5e569-37ea-4982-a706-573c3a626778.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "chore: use context passed handler",
+  "packageName": "@fluentui/react-button",
+  "email": "chassunc@microsoft.com",
+  "dependentChangeType": "none"
+}
```

Comment:
Is this required? There're no modifications on `react-button` anymore.

Reply:
Removed